test/Microsoft.AspNetCore.Mvc.DataAnnotations.Test/Internal/DataAnnotationsMetadataProviderTest.cs

```diff
@@ -180,14 +194,21 @@ public void CreateDisplayMetadata_DisplayAttribute_NameFromResources()
 
             // Assert
             Assert.Equal("name from resources", context.DisplayMetadata.DisplayName());
+
+            if (stringLocalizerFactory != null)
+            {
+                stringLocalizerFactory.Verify(s => s.Create(It.IsAny<Type>()));
+            }
         }
 
+
         // This is IMPORTANT. Product code needs to use GetDescription() instead of .Description. It's easy to regress.
-        [Fact]
-        public void CreateDisplayMetadata_DisplayAttribute_DescriptionFromResources()
+        [Theory]
+        [MemberData(nameof(LocalizerFactoryMock))]
+        public void CreateDisplayMetadata_DisplayAttribute_DescriptionFromResources(Mock<IStringLocalizerFactory> stringLocalizerFactory)
```

Comment:
You have a test like this for `Name` and `Description`.  Does one exist for Prompt?